app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -155,4 +169,47 @@ private String getKeyHash(){
             return "";
         }
     }
+
+    private void deleteExistingApks() {
+
+        /**
+         * Skip checking & deleting APK files in the first start-up of application
```

Comment:
Remove this check. Only perform the deletion if `newVersionCode < 2000009`.

Reply:
Do you mean this check should be removed? `if (UserPrefsHelper.getLocale(context) == null)`
That's for preventing NPE.

Reply:
Yes, because the method should not trigger if the app is not being upgraded from a previous version.